redbot/core/core_commands.py

```diff
@@ -1356,41 +1355,81 @@ class Core(commands.Cog, CoreLogic):
 
         description = _("Sent by {} {}").format(author, source)
 
-        if isinstance(author, discord.Member):
-            colour = author.colour
-        else:
-            colour = discord.Colour.red()
+        destinations = await ctx.bot.get_owner_notification_destinations()
 
-        if await ctx.embed_requested():
-            e = discord.Embed(colour=colour, description=message)
-            if author.avatar_url:
-                e.set_author(name=description, icon_url=author.avatar_url)
+        if not destinations:
+            await ctx.send(_("I've been configured not to send this anywhere."))
+            return
+
+        successful = False
+
+        for destination in destinations:
+
+            is_dm = isinstance(destination, discord.User)
+            send_embed = None
+
+            if is_dm:
+                send_embed = await ctx.bot.db.user(destination).embeds()
             else:
-                e.set_author(name=description)
-            e.set_footer(text=footer)
+                if not destination.permissions_for(destination.guild.me).send_messages:
+                    continue
+                if destination.permissions_for(destination.guild.me).embed_links:
+                    send_embed = await ctx.bot.db.guild(destination.guild).embeds()
+                else:
+                    send_embed = False
+
+            if send_embed is None:
+                send_embed = await self.db.embeds()
```

Comment:
```suggestion
                send_embed = await ctx.bot.db.embeds()
```
Seem to have missed a bit here.